tubular/mixins.py

```diff
@@ -187,13 +140,14 @@ class attribute, indicates whether transformer has been converted to polars/pand
 
     polars_compatible = False
 
-    def check_weights_column(self, X: pd.DataFrame, weights_column: str) -> None:
+    @nw.narwhalify
+    def check_weights_column(self, X: FrameT, weights_column: str) -> None:
         """Helper method for validating weights column in dataframe.
 
         Args:
         ----
-            X (pd.DataFrame): df containing weight column
-            weights_column (str): name of weight column
+            dfs: df containing weight column
```

Comment:
shouldnt this be X? realise we didn't update docstrings in last PR to reflect that X can be a pd or pl Dataframe.

Reply:
ah thanks, updated here https://github.com/lvgig/tubular/pull/331/commits/161db20158b881dc1d23b4f5703422cf0e30d6a4